flow_results/testsettings.py

```diff
@@ -0,0 +1,3 @@
+from flow_results.settings import *  # noqa: F403, F401
+
+PROMETHEUS_EXPORT_MIGRATIONS = False
```

Comment:
@Hlamallama this is what was causing the docker build to attempt to connect to the DB, it shouldn't need to for collectstatic. To disable I created a testsettings.py file and used that when running the coilectstatic command in the Dockerfile